PhysicsTools/PatAlgos/plugins/SlimCaloJetSelector.h

```diff
@@ -0,0 +1,84 @@
+#ifndef PhysicsTools_PatAlgos_SlimCaloJetSelector_h
+#define PhysicsTools_PatAlgos_SlimCaloJetSelector_h
+
+#include "FWCore/Framework/interface/stream/EDFilter.h"
+#include "DataFormats/Common/interface/RefVector.h"
+
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/ObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleElementCollectionSelector.h"
+
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include <vector>
+
+class SlimCaloJetSelector : public edm::stream::EDFilter<> {
```

Comment:
this makes sense if it will never ever include any dependence on PAT classes.
Would it or would it not?